content/working_with/console/widgets/userManagement.md

```diff
@@ -23,9 +36,56 @@ The hamburger menu on the right of every tenant allows performing the following
 * **Deleting the user** - possible only if the user does not belong to any groups, assigned to any tenants and is the creator of any resources on the manager. 
  
 Also, using the “Add” button on the right top corner of the widget, you will be able to create new users.
-Please notice that if you choose to  authenticate the users in front of an external user management system, you will not be able to create or delete the users in cloudify, nor to assigned them to Cloudify user groups,  to prevent conflicts between the two systems which might cause security problems. 
+Please notice that if you choose to authenticate the users in front of an external user management system, you will not be able to create or delete the users in cloudify, nor to assigned them to Cloudify user groups,  to prevent conflicts between the two systems which might cause security problems. 
+
+#### Adding Users
+
+1. Click **Add** button in the widget.
+2. Specify a name for the user.   
+   The name must contain at least five alphanumeric characters and begin with a letter. It can also include the following characters `-`, `_`, or `.`.
```

Comment:
```suggestion
   The name must contain at least five alphanumeric characters and begin with a letter. It can also include the following characters: `-`, `_` or `.`.
```